[rospy.client][INFO] 2021-05-02 14:53:49,421: init_node, name[/servo_commands], pid[24918]
[xmlrpc][INFO] 2021-05-02 14:53:49,421: XML-RPC server binding to 0.0.0.0:0
[xmlrpc][INFO] 2021-05-02 14:53:49,421: Started XML-RPC server [http://ubuntu:36999/]
[rospy.impl.masterslave][INFO] 2021-05-02 14:53:49,421: _ready: http://ubuntu:36999/
[rospy.init][INFO] 2021-05-02 14:53:49,425: ROS Slave URI: [http://ubuntu:36999/]
[rospy.registration][INFO] 2021-05-02 14:53:49,426: Registering with master node http://localhost:11311
[xmlrpc][INFO] 2021-05-02 14:53:49,427: xml rpc node: starting XML-RPC server
[rospy.init][INFO] 2021-05-02 14:53:49,526: registered with master
[rospy.rosout][INFO] 2021-05-02 14:53:49,526: initializing /rosout core topic
[rospy.rosout][INFO] 2021-05-02 14:53:49,529: connected to core topic /rosout
[rospy.simtime][INFO] 2021-05-02 14:53:49,532: initializing /clock core topic
[rospy.simtime][INFO] 2021-05-02 14:53:49,538: connected to core topic /clock
[rospy.internal][INFO] 2021-05-02 14:53:49,559: topic[/vesc/low_level/ackermann_cmd_mux/output] adding connection to [http://ubuntu:43793/], count 0
[rospy.internal][INFO] 2021-05-02 14:53:49,805: topic[/rosout] adding connection to [/rosout], count 0
[rospy.internal][INFO] 2021-05-02 14:53:51,582: topic[/clock] adding connection to [http://ubuntu:46441/], count 0
[rospy.core][INFO] 2021-05-02 14:54:24,161: signal_shutdown [signal-2]
[rospy.internal][WARNING] 2021-05-02 14:54:24,164: Unknown error initiating TCP/IP socket to ubuntu:36657 (http://ubuntu:43793/): Traceback (most recent call last):
  File "/opt/ros/melodic/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 562, in connect
    self.read_header()
  File "/opt/ros/melodic/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 657, in read_header
    self._validate_header(read_ros_handshake_header(sock, self.read_buff, self.protocol.buff_size))
  File "/opt/ros/melodic/lib/python2.7/dist-packages/rosgraph/network.py", line 357, in read_ros_handshake_header
    d = sock.recv(buff_size)
error: [Errno 104] Connection reset by peer

[rospy.internal][INFO] 2021-05-02 14:54:24,164: topic[/vesc/low_level/ackermann_cmd_mux/output] removing connection to http://ubuntu:43793/
[rospy.internal][INFO] 2021-05-02 14:54:24,213: topic[/rosout] removing connection to /rosout
[rospy.internal][INFO] 2021-05-02 14:54:24,213: topic[/clock] removing connection to http://ubuntu:46441/
[rospy.impl.masterslave][INFO] 2021-05-02 14:54:24,213: signal-2
[rospy.core][INFO] 2021-05-02 14:54:24,839: signal_shutdown [atexit]
